### PR TITLE
Added conda instructions and environment file

### PR DESCRIPTION
**Pull request by aashish24**

### CODE REVIEW

**Comment by aashish24** [comment]
@matthewma7 @manthey please review.

**Comment by jbeezley** [comment]
The encryption key is used to store user credentials for external WMS servers.  It avoids having to store those credentials in plain text in the database.  Generating that config option should probably be added somehow to the production deployment.

It is a little tricky because you don't want to change an existing encryption key if you are reusing an existing mongo database that already contains credentials.  I think it should come from some private external setting or file that is used by a specific deployment rather than just generated randomly.

**Comment by aashish24** [comment]
thanks @matthewma7 so I guess that was the missing piece. Can I remove instructions for running pip install -r requirements.txt then?

**Comment by matthewma7** [comment]
@jbeezley Thank you for explaining the encryption.

**Comment by matthewma7** [comment]
@aashish24 Yes, the instructions **pip install -r requirements.txt** can be removed.

**Comment by aashish24** [comment]
@matthewma7 this is ready for your review now.

File: docs/installation.rst [hunk 1]
```diff
@@ -45,54 +45,106 @@ Fedora 22
 .. _installing node.js on Red Hat: https://nodejs.org/en/download/package-manager/#enterprise-linux-and-fedora
 
 
+Conda
+^^^^^^^^^
```
Comment (matthewma7) [inline docs/installation.rst]: It's working fine but probably should remove the extra characters.

**Comment by aashish24** [comment]
@matthewma7 I forced-push in this case so rst for conda is good now.

File: docs/installation.rst [hunk 2]
```diff
@@ -45,54 +45,106 @@ Fedora 22
...
-- Install girder_db_items plugin into the Girder plugins directory.
+- Install girder_ktile plugin.
 
 ::
 
-   cd GIRDER_DIR/plugins
-   git clone https://github.com/OpenGeoscience/girder_db_items database_assetstore
+    git clone https://github.com/OpenGeoscience/girder_ktile
+    girder-install plugin -s girder_ktile
+    girder-install web --dev --plugins girder_ktile
```
Comment (matthewma7) [inline docs/installation.rst]: Technically, when we build Minerva, it's dependencies (girder_ktile and database_assetstore) will be built too. So, do you think we need this step?

File: docs/installation.rst [hunk 3]
```diff
@@ -45,54 +45,106 @@ Fedora 22
...
+    git clone https://github.com/Kitware/minerva.git
+    cd minerva
+    conda env create -f conda_env.yml python=2.7
+    source activate minerva-dev
+
+- Run mongo daemon
+
+::
+
+    mongod &
```
Comment (matthewma7) [inline docs/installation.rst]: With a clean Ubuntu, I need to create a directory for mongo frist then run
**mkdir mongo_data
mongod --dbpath mongo_data &**

File: docs/installation.rst [hunk 4]
```diff
@@ -136,6 +191,12 @@ Notes:
     cd GIRDER_DIR
```
Comment (matthewma7) [inline docs/installation.rst]: Do we want to remove this alternative method?

File: docs/installation.rst [hunk 5]
```diff
@@ -45,54 +45,106 @@ Fedora 22
...
+Conda installation and conda environment setup should be the first step in
+creating a development environment for Minerva.
+
+- Install Conda (https://conda.io/miniconda.html)
+
+- Create minerva development environment
+
+::
+
+    mkdir geoviz
```
Comment (matthewma7) [inline docs/installation.rst]: Is this geoviz directory necessary?

File: docs/installation.rst [hunk 6]
```diff
@@ -45,54 +45,106 @@ Fedora 22
...
+::
+
+    mongod &
+
+- Install Girder
+
+::
+
+    cd ..
+    git clone https://github.com/girder/girder.git
```
Comment (matthewma7) [inline docs/installation.rst]: Should we checkout to girder 2.x-maintenance branch?

File: docs/installation.rst [hunk 7]
```diff
@@ -112,6 +164,9 @@ Notes:
     cd GIRDER_DIR
```
Comment (matthewma7) [inline docs/installation.rst]: Should we remove this method?

**Comment by matthewma7** [comment]
Hi @aashish24 I added more comment, some are not preventing the installing. 
It's not related to this PR, but some wording and structure can be improved, I'll create a PR for further improvement. 
Overall, this document worked for me to create the environment for Minerva.

File: docs/installation.rst (hunk 2 above)
Comment (aashish24) [reply docs/installation.rst]: that is a good point does it work all the time? I always had this issue where in some cases it did not. Did you experience the same?

File: docs/installation.rst (hunk 3 above)
Comment (aashish24) [reply docs/installation.rst]: Fixed

File: docs/installation.rst (hunk 2 above)
Comment (matthewma7) [reply docs/installation.rst]: Yes, I think it always works for me. In fact, when I am testing this, I only run the Minerva one, and it worked good.

**Comment by aashish24** [comment]
thanks @matthewma7

**Pull request merged**